src/screens/LearningScreen/components/FlashCard.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import {
+  getExpressions,
+  getTargetLemma,
+  getSourceLemma,
+} from '../../../api/getters';
+
+const styles = StyleSheet.create({
+  card: {
+    margin: 5,
+    backgroundColor: 'white',
+  },
+});
+
+const FlashCard = ({ word, isRevealed, isExpanded, children }) => (
+  <View style={styles.card}>
+    <Text>{word.targets ? getTargetLemma(word) : null}</Text>
```

Comment:
If you can't get a target lemma or source, should anything be displayed here for this flash card or should that be considered an error and another piece of UI displayed instead?